components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/ha/HACoordinationRecordTableHandler.java

```diff
@@ -213,6 +213,10 @@ public void setAsActive() throws ConnectionUnavailableException {
         }
     }
 
+    public void setPassive() {
+        this.isActiveNode = false;
```

Comment:
Shall we also reset the queues when moving to passive, if it had something them it can be a problem. 
Lets reset all the queues in each section like passiveNodeProcessedEvents, eventQueue, passiveNodeBufferedEvents, etc

Reply:
Fixed with a39bc872d9840cc87ae30ea135947a5ac2f623c8